content/development/clients/index.en.yaml

```diff
@@ -0,0 +1,54 @@
+contribute: true
+disclaimer: true
+title: Clients
+description: Download links for various Ethereum Classic clients and deployment tools
+intro: Clients are used to sync with the Ethereum Classic network and verify blocks. Most users won't need to run a full node, but they allow those who do to trustlessly interact with the network. Ethereum Classic is scheduled for a network update in June, 2020. Here are [Ethereum Classic Client Upgrade Instructions](https://hackmd.io/IPA7zPwpQDy-1U0reeppcQ).
+content:
+  -
+    configRef: clients
```

Comment:
You can refer to sibling files using `configRef` or `itemsRef` to keep the file size down.